deploy/kubernetes/configmap.yaml

```diff
@@ -174,7 +174,7 @@ data:
     # no baseline rule will be created and the cluster will assume the default
     # behavior as specified by the backend. The option is only supported on
     # Policy API.
-    # Choices: <None> allow_cluster allow_namespace allow_namespace_strict
+    # Choices: allow_cluster allow_namespace allow_namespace_strict None NONE none
```

Comment:
`None NONE none`: I guess we can just list one

Reply:
These three values are from auto generated configmap yaml based recent ncp changes.So, it might be to keep as is